app/models/teacher_training_adviser/steps/identity.rb

```diff
@@ -22,12 +24,16 @@ def export
       super.except("sub_channel_id")
     end
 
+    def latest_privacy_policy
+      @latest_privacy_policy ||= GetIntoTeachingApiClient::PrivacyPoliciesApi.new.get_latest_privacy_policy
+    end
+
     def reviewable_answers
       super
         .tap { |answers|
           answers["name"] = "#{answers['first_name']} #{answers['last_name']}"
         }
-        .without("first_name", "last_name", "channel_id", "sub_channel_id")
+        .without("first_name", "last_name", "channel_id", "sub_channel_id", "accepted_policy_id")
```

Comment:
Nitpick: Might be clearer to use shorthand here:
``` Ruby 
 %w[first_name last_name channel_id sub_channel_id accepted_policy_id]
```

Reply:
I'll merge this as-is for now as I want a full day of it being out to keep an eye on it (as its Friday) but I'll make that change when I'm back in this model, which I have a ticket around DRYing up in general so should be very shortly.